### PR TITLE
Update code readability

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@extremeheat

File: src/server.js [hunk 1]
```diff
@@ -79,28 +86,31 @@ class Server extends EventEmitter {
...
   }
 
   async listen (host = this.options.host, port = this.options.port) {
     this.raknet = new this.RakServer({ host, port }, this)
+
     try {
       await this.raknet.listen()
     } catch (e) {
-      console.warn(`Failed to bind server on [${this.options.host}]/${this.options.port}, is the port free?`)
-      throw e
```
Comment (extremeheat) [inline src/server.js:102]: This should be throwing the actual underlying exception. "Failed to bind" is the result not error. What you can do for the message above is make it a connection log (conLog) message.

File: src/server.js [hunk 2]
```diff
@@ -79,28 +86,32 @@ class Server extends EventEmitter {
...
   }
 
   async listen (host = this.options.host, port = this.options.port) {
     this.raknet = new this.RakServer({ host, port }, this)
+
     try {
       await this.raknet.listen()
     } catch (e) {
-      console.warn(`Failed to bind server on [${this.options.host}]/${this.options.port}, is the port free?`)
+      console.warn(`Failed to bind server on [${this.options.host}]/${this.options.port}], is the port free?`)
```
Comment (extremeheat) [inline src/server.js]: The bracket should be around the hostname. It's intended to encapsulate an IPv6 address.

Comment (ghost) [reply src/server.js]: oops, i didn't see the other bracket after the hostname

**Review by extremeheat — APPROVED** [review]

**Comment by extremeheat** [comment]
Thanks for the contribution. The spacing is unnecessary imo, but otherwise LGTM.

**Pull request merged**